src/main/java/com/softserve/paymentservice/controller/PaymentController.java

```diff
@@ -0,0 +1,83 @@
+package com.softserve.paymentservice.controller;
```

Comment:
Nit: as you write payment service specifically for kick scooter application, it should be addressed in your package name.
group: `com.softserve.kickscooter`
artifact: `kick-scooter-payment`
package: `com.softserve.kickscooter.payment.controller`